### ISSUE TITLE
aws COMMAND returns message Invalid Region in ARN if in non-default region

### ISSUE THREAD

**Issue by januszm** [aws/aws-cli#3175]
**TLDR;**
When invoking an `aws` command for resources in non-default region, a cryptic "Invalid Region in ARN" message is shown. Perhaps we could make it more verbose and return something like "resource region does not match configured command region" etc.

----

Hello, I'm trying to run the AWS ECS Fargate task using CLI using the following command
(`awscli` version 1.14.50 installed via `pip`)

```
$ aws ecs run-task --cluster "arn:aws:ecs:us-east-1:ACCOUNTID:cluster/MY-FARGATE-CLUSTER" --task-definition "arn:aws:ecs:us-east-1:ACCOUNTID:task-definition/MY-ECSTASK:2" --launch-type FARGATE --network-configuration "awsvpcConfiguration={subnets=[subnet-12345],securityGroups=[sg-23456,sg-56789],assignPublicIp=ENABLED}" --count 1

An error occurred (InvalidParameterException) when calling the RunTask operation: Invalid Region in ARN
```

When I use the short forms for cluster and task-definition, I get this error message:

```
$ aws ecs run-task --cluster "MY-FARGATE-CLUSTER" --task-definition "MY-ECSTASK:2" ...

An error occurred (ClusterNotFoundException) when calling the RunTask operation: Cluster not found.
```

**UPDATE**

This issue was caused by the fact that the default region in my `aws cli` configuration was set to `us-east-2` and I was trying to run a FARGATE task in region `us-east-1`. The error message is a bit confusing.

**Comment by joguSD** [comment]
Unfortunately this error is coming back from the ECS service and we won't be able to change it from the CLI side.

**Comment by yorjaggy** [comment]
This fixed my problem, i was having a diff region name 🤦🏽 on my ~/.aws/config file
> This issue was caused by the fact that the default region in my aws cli configuration was set to us-east-2 and I was trying to run a FARGATE task in region us-east-1. The error message is a bit confusing.

Thanks!
